### DISCUSSION TITLE
中国国内的官网怎么报404？什么时候能好？

### DISCUSSION THREAD

**Discussion by Wwh-SoEximious** [youzan/vant#12835] [Q&A] [answered]
Vant 3 的官网中国访问会报404

How do you report 404 on the official website in China?

**Comment by chenjiahan** [comment, marked as answer]
参考 https://github.com/youzan/vant/issues/12832

**Reply by Wwh-SoEximious** [reply]
看到了，感谢

**Comment by Wwh-SoEximious** [comment]
想起来，这是我第一次在GitHub上提问，当时还不怎么会用，现在想想也是有进步了😀。抱抱自己，真棒！👍

**Reply by inottn** [reply]
👍
